modules/vue-router-simple/concepts/Pages.js

```diff
@@ -128,11 +128,12 @@ import { defineAsyncComponent } from 'vue'
 export { default as App } from '<%= app %>'
 import plugins from './vueSimpleRouter/plugins'
 
-export const layouts = Object.values(plugins).reduce((layouts, plugin) => plugin(layouts), [
+export const layouts = Object.values(plugins).reduce((layouts, plugin) => plugin(layouts,true), [
```

Comment:
Here you will pass more readable confiratuon object

`plugin(layouts, { isLayout: true }`